examples/build.py

```diff
@@ -43,16 +31,26 @@ def listify(a):
 
 coords_ids = listify(np.arange(N))
 coords = np.random.randn(N, 10).tolist()
-pct_var = (1/np.exp(np.arange(10))).tolist()
+pct_var = 1/np.exp(np.arange(10))
+
 
 md_headers = ['SampleID', 'DOB', 'Strings']
 metadata = []
 for _id in coords_ids:
-    metadata.append([_id, ''.join(sample(categories, 1)), ''.join(choice(
+    metadata.append([_id, ''.join(sample(set(categories), 1)), ''.join(choice(
```

Comment:
out of scope here, but might be good to have a mixed string/character column as well for testing using build.py